ceph/rados/core_workflows.py

```diff
@@ -3173,3 +3173,45 @@ def get_osd_perf_dump(self, osd_id):
         """
         base_cmd = f"ceph tell osd.{osd_id} perf dump"
         return self.run_ceph_command(cmd=base_cmd)
+
+    def get_available_path_list(self, node: str, device_type: str):
+        """
+        Method returns the available SSD list in the provided node.
+            Args:
+                 rados_object: RadosOrchestrator class object
+                 node: node name
+
+             Returns: Returns the available device path list of the node.
+
+        """
+        ssd_paths = []
+        available_device_list = self.get_orch_device_list(node)
+        for path_list in available_device_list[0]["devices"]:
+            if path_list["human_readable_type"] != device_type:
+                continue
+            if (
+                path_list["human_readable_type"] == device_type
+                and path_list["available"] is True
+            ):
+                ssd_paths.append(path_list["path"])
+        return ssd_paths
+
+    def is_path_available(self, node, path):
```

Comment:
this method can be removed and the above method ` get_available_path_list ` can be used to check if the device is free.

Reply:
Removed the is_path_available method.